client/src/js/app/tasks.service.js

```diff
@@ -0,0 +1,117 @@
+angular
+  .module('toDoList')
+  .service('tasksService', tasksService);
+
+tasksService.$inject = ['$resource'];
+
+function tasksService($resource) {
+  var serv = this;
+  serv.tasks = [];
+  serv.addTask = addTask;
+  serv.deleteTask = deleteTask;
+  serv.loadTasks = loadTasks;
+  serv.decodeDate = decodeDate;
+  serv.editTaskName = editTaskName;
+  serv.editTaskDate = editTaskDate;
+  serv.editTaskStatus = editTaskStatus;
+  serv.userUri=undefined;
```

Comment:
I think it might be helpful to use jslint in your project to be consistent with code style (spaces and other things)